ome_zarr.py

```diff
@@ -241,16 +252,10 @@ def load_ome_zarr(self):
                 str(c[0]) + (" (+ %s)" % c[-1] if c[-1] != c[0] else "")
                 for c in data.chunks
             ]
-            print(
-                "resolution",
-                resolution,
-                "shape (t, c, z, y, x)",
-                data.shape,
-                "chunks",
-                chunk_sizes,
-                "dtype",
-                data.dtype,
-            )
+            LOGGER.info("resolution: %s", resolution)
+            LOGGER.info(" - shape (t, c, z, y, x) = %s", data.shape)
```

Comment:
What do you think about making this into a single log statement containing `\n`? If in future these logs are automatically parsed it'll be easier.

Reply:
I personally find the indentation then harder to read. I assume if we ever want these parsed we'll use a json or yaml output, no?

Reply:
If our other projects are anything to go by we'll just end up parsing the unmodified output 😀. Merging anyway.